syntax = "proto3";

package armonik.api.grpc.v1.worker;

import "objects.proto";
import "worker_common.proto";

option csharp_namespace = "ArmoniK.Api.gRPC.V1.Worker";
option java_package = "fr.aneo.armonik.api.grpc.v1.worker";

service Worker {
  rpc Process(ProcessRequest) returns (ProcessReply);
  rpc HealthCheck(Empty) returns (HealthCheckReply);
}
